src/js/tabs/gps.js

```diff
@@ -193,7 +192,8 @@ gps.initialize = async function (callback) {
             const healthyArray = ['gnssHealthyUnknown', 'gnssHealthyHealthy', 'gnssHealthyUnhealthy', 'gnssHealthyUnknown'];
             let alt = FC.GPS_DATA.alt;
 
-            $('.GPS_info td.fix').html(getColorYesNo(FC.GPS_DATA.fix));
+            $('.GPS_info span.fix').text((FC.GPS_DATA.fix) ? i18n.getMessage('gpsFixTrue') : i18n.getMessage('gpsFixFalse'));
```

Comment:
Parenthesis not needed
 
```suggestion
            $('.GPS_info span.fix').text(FC.GPS_DATA.fix ? i18n.getMessage('gpsFixTrue') : i18n.getMessage('gpsFixFalse'));
```